package.json

```diff
@@ -6,15 +6,11 @@
   "dependencies": {
     "nan": "~1.5.0"
   },
-  "os": [
-    "darwin"
```

Comment:
repeating the comment I made about this in #66 

Why remove this? It's what allows fsevents to be included as an optional dependency on a cross-platform module.

The gyp configuration for other OSes did get smoothed out at some point, but I don't see why we wouldn't still specify the compatible OS here.